services/ui-src/src/libs/faq/faqContent.tsx

```diff
@@ -555,6 +555,62 @@ export const oneMACFAQContent: FAQContent[] = [
           </div>
         ),
       },
+      {
+        anchorText: "medicaid-spa-withdraw-rai-response",
+        isOpen: false,
+        question: "How do I Withdraw a Formal RAI Response for a Medicaid SPA?",
+        answerJSX: (
+          <div>
+            <p>
+              If a state wishes to withdraw a Formal RAI Response, the state
+              must first contact their CMS Point of Contact so the action can be
+              enabled.
+            </p>
+            <ul>
+              <li>
+                As a CMS user, log in to OneMAC and select the link to the SPA
+                ID from the dashboard
+              </li>
+              <li>
+                Then, under Package Actions, select the Enable Formal RAI
+                Response Withdraw link, and then Select Submit.
+              </li>
+            </ul>
+            <p>
+              After receiving confirmation from your CMS Point of Contact that
+              the Withdraw Formal RAI Response feature has been enabled, locate
+              and select the Medicaid SPA submission package.
+            </p>
+            <p>
+              The package status remains as Under Review and a substatus of RAI
+              Response Withdraw Enabled will be reflected below the status for
```

Comment:
the substatus is: Withdraw Formal RAI Response Enabled  -- I know, the AC of the story is wrong...